project_resource_calendar/i18n/fr.po

```diff
@@ -745,17 +777,14 @@ msgstr "Dimanche"
 
 #. module: project_resource_calendar
 #: model:ir.model.fields,field_description:project_resource_calendar.field_calendar_event_event_task_id
-#: model:ir.model.fields,field_description:project_resource_calendar.field_doublebooking_validation_wiz_task_id
 msgid "Task"
 msgstr "Tâche"
 
 #. module: project_resource_calendar
-#: code:addons/project_resource_calendar/models/calendar_event.py:250
+#: code:addons/project_resource_calendar/models/calendar_event.py:244
 #, python-format
-msgid ""
-"The attendee %s cannot be double-booked with any overlapping meetings or "
-"events."
-msgstr "Le participant, %s, est déjà réservé pour la période spécifiée."
+msgid "The %s: %s From:%s To:%s\n"
+msgstr "The %s: %s From:%s To:%s\n"
```

Comment:
translation?

Reply:
I wasn't originally in the POT file I downloaded from Odoo to add new translations.